Reflections/Week_10_Reflection.md

```diff
@@ -0,0 +1,14 @@
+1. What have you accomplished this week? (Please list each accomplishment, and explain)
+  - Got the most basic linear actuator made to get an understanding of what I need. 
+  - Updated some of the files throughout the project
+  - Have a longboard to test if the system currently has the power to move it as well.
```

Comment:
Testing your project on two different sized boards seems like a great idea for seeing how it performs with the two different weights of the boards.